src/components/Modals/CreateOrder/index.jsx

```diff
@@ -0,0 +1,163 @@
+import { useHistory } from 'react-router-dom';
+import ordersService from 'Api/orders.service';
+import categoriesService from 'Api/categories.service';
+import Spinner from 'Components/Utils/Spinner';
+import ErrorModal from 'Components/Modals/ErrorModal';
+import 'react-toastify/dist/ReactToastify.css';
+
+const CreateOrder = () => {
+  const [categories, setCategories] = useState([]);
+  const [description, setDescription] = useState('');
+  const [options, setOptions] = useState([]);
+  const [modalIsOpen, setIsOpen] = useState(false);
+  const [invalid, setInvalid] = useState(false);
+  const [errorMessage, setError] = useState('');
+  const [loading, setLoading] = useState(false);
+  const userLoggedIn = useSelector((state) => state.logUser.loggedIn);
+  const helpeeId = useSelector((state) => state.logUser.data.id);
+  const history = useHistory();
+
+  const getCategories = async () => {
+    const opts = [];
+    try {
+      const categoriesJson = await categoriesService.getCategories();
+      categoriesJson.forEach((opt) => {
+        opts.push({
+          value: opt.id,
+          label: opt.description,
+        });
+      });
+    } catch (error) {
+      notifyError(error);
+    }
+    setOptions(opts);
+  };
+
+  const newOrder = async () => {
+    let errorFlag = false;
+    const title = 'Titulo del pedido';
+    if (categories.length === 0) {
+      notifyError('Debe seleccionar al menos una categoría');
+      errorFlag = true;
+    }
+    if (!description) {
+      notifyError('Descripción es obligatoria');
+      errorFlag = true;
+    }
+    if (!errorFlag) {
+      try {
+        setLoading(true);
+        await ordersService.create({
+          title,
+          helpeeId,
+          categories,
+          description,
+        });
+      } catch (error) {
+        setLoading(false);
+        setInvalid(true);
+        setError('Ocurrío un error intentando comunicarse con el servidor');
+        errorFlag = true;
+      }
+    }
+    if (!errorFlag) {
+      setLoading(false);
+      notifySuccess('Su pedido fue registrado con exito');
+      closeModal();
+    }
+  };
+
+  const openModal = () => {
+    // Set an empty description
+    setDescription('');
+    setIsOpen(true);
+  };
+
+  const closeModal = () => {
+    // Set an empty description
+    setDescription('');
+    setIsOpen(false);
+  };
+
+  const handleChange = (selectedOptions) => {
+    if (selectedOptions) {
+      setCategories(Array.from(selectedOptions, (option) => ({ id: option.value })));
+    } else {
+      setCategories([]);
+    }
+  };
+
+  const notifyError = (errorMsg) => toast.error(errorMsg);
+
+  const notifySuccess = (successMsg) => toast.success(successMsg);
+
+  useEffect(() => {
+    if (!userLoggedIn) {
+      history.push('/login');
+    } else {
+      getCategories();
+    }
+  }, []);
+
+  return (
+    <div>
+      <Button disabled={loading} secondary onClick={openModal} label="Nuevo Pedido" />
```

Comment:
Agregar pedido es la opcion principal par el usuario el boton deberia ser primario

```suggestion
      <Button disabled={loading} primary onClick={openModal} label="Nuevo Pedido" />
```